pkg/tbtc/node.go

```diff
@@ -684,7 +684,64 @@ func (n *node) handleMovedFundsSweepProposal(
 	startBlock uint64,
 	expiryBlock uint64,
 ) {
-	// TODO: Implement
```

Comment:
Let's also remove the `// TODO: Uncomment when moving funds support is implemented.` comment from `processCoordinationResult` function.

Reply:
Done in 8979f41c3a6837c1040849799efe55ac10bda595.